### PR TITLE
[IOPAE-1376] Add ci missing stuff

### PR DESCRIPTION
**Pull request by giamma1295**
Add Ci and missing configs.
#### List of Changes
- Added PR Template
- Added Labeler action and related configs
- Added Code-Review Action
- Update turbo 1.2.x -> 2.x

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by rodoviti — APPROVED** [review]
LGTM

**Review by giuseppedipinto — APPROVED** [review]

File: .github/workflows/code-review.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      - apps/**
+      - packages/**
+      - package.json
+  workflow_dispatch:
+
+jobs:
+  code_review:
+    runs-on: ubuntu-latest
+
+    steps:
```
Comment (Krusty93) [inline .github/workflows/code-review.yaml:19]: I guess you can use [the available template](https://github.com/pagopa/dx/blob/main/.github/workflows/js_code_review.yaml)?

Comment (giamma1295) [reply .github/workflows/code-review.yaml:19]: The template works only with turbo 1.x @lucacavallaro, we need to downgrade turbo in order to use the dx template?

Comment (giamma1295) [reply .github/workflows/code-review.yaml:19]: @lucacavallaro i dont have the permission on dx repo , unfortunately i cant open any pr

File: .github/workflows/code-review.yaml [hunk 2]
```diff
@@ -35,8 +35,18 @@ jobs:
...
+        run: |
+          TURBO_VERSION=$(yarn turbo --version)
+          echo "Turbo version: $TURBO_VERSION"
+          if [[ "$TURBO_VERSION" =~ ^1\..* ]]; then
+            TURBO_CACHE_PATH="node_modules/.cache/turbo"
+          else
+            TURBO_CACHE_PATH=".turbo"
+          fi
+          echo "TURBO_CACHE_PATH=$TURBO_CACHE_PATH" >> $GITHUB_ENV
+          echo "Using cache path: $TURBO_CACHE_PATH"
```
Comment (lucacavallaro) [inline .github/workflows/code-review.yaml:49]: maybe can we explicitly set `--cache-dir` (or `TURBO_CACHE_DIR`) to maintain the same directory

Comment (lucacavallaro) [reply .github/workflows/code-review.yaml:49]: https://turbo.build/repo/docs/reference/run#--cache-dir-path

File: .github/workflows/code-review.yaml (hunk 1 above)
Comment (giamma1295) [reply .github/workflows/code-review.yaml:19]: Pr opened: https://github.com/pagopa/dx/pull/100

File: .github/workflows/code-review.yaml (hunk 2 above)
Comment (giamma1295) [reply .github/workflows/code-review.yaml]: I've tested forcing the cache dir using the env and it works!
Test using turbo 1.x.x and cache dir like in 2.x.x: https://github.com/pagopa/io-ipatente/actions/runs/10844985008/job/30094904145

File: .github/workflows/code-review.yaml (hunk 1 above)
Comment (giamma1295) [reply .github/workflows/code-review.yaml]: updated Dx template and use it here!

File: .github/workflows/code-review.yaml (hunk 2 above)
Comment (giamma1295) [reply .github/workflows/code-review.yaml]: done in dx template

**Review by gunzip — APPROVED** [review]

**Review by krusty93 — APPROVED** [review]

**Pull request merged**